ECE_VERSION

```diff
@@ -1 +1 @@
-3.6.0
+3.6.1
```

Comment:
Wouldn't it be 3.7.0? Since 3.6.1 is based off of ms-92.

Reply:
Good question. I will change it to 3.7.0.